### PR TITLE
popover on news page

### PR DESCRIPTION
**Pull request by mlevoy**
### What github.com/wevote/WebApp/issues does this fix?
https://github.com/wevote/WebApp/issues/3125
### Changes included this pull request?
StickyPopover added on news page posters of new opinions

### CODE REVIEW

File: src/js/components/Activity/ActivitySpeakerCard.jsx [hunk 1]
```diff
@@ -71,31 +73,46 @@ class ActivitySpeakerCard extends Component {
...
-        ) : (
-          <ProfileAvatar>
-            <ActivityImage src={cordovaDot(avatarGenericIcon)} alt={`${speakerName}`} />
-          </ProfileAvatar>
-        )}
+        <StickyPopover
+          delay={{ show: 700, hide: 100 }}
+          popoverComponent={organizationPopoverCard}
+          placement="auto"
+          id="positions-organization-popover-trigger-click-root-close"
```
Comment (DaleMcGrew) [inline src/js/components/Activity/ActivitySpeakerCard.jsx:92]: Hi @mlevoy I will fix this, but id's are supposed to be unique when on the same page. I have also been shifting towards camelCase ids.

File: src/js/components/Activity/ActivitySpeakerCard.jsx [hunk 2]
```diff
@@ -7,8 +7,10 @@ import { cordovaDot } from '../../utils/cordovaUtils';
 import LoadingWheel from '../LoadingWheel';
 import { renderLog } from '../../utils/logging';
 import OpenExternalWebSite from '../Widgets/OpenExternalWebSite';
+import OrganizationPopoverCard from '../Organization/OrganizationPopoverCard';
 import { numberWithCommas, timeFromDate } from '../../utils/textFormat';
 import VoterStore from '../../stores/VoterStore';
+import StickyPopover from '../Ballot/StickyPopover';
```
Comment (DaleMcGrew) [inline src/js/components/Activity/ActivitySpeakerCard.jsx:13]: Please try to alphabetize new imports when possible.

**Comment by DaleMcGrew** [comment]
👍

File: src/js/components/Activity/ActivitySpeakerCard.jsx (hunk 1 above)
Comment (mlevoy) [reply src/js/components/Activity/ActivitySpeakerCard.jsx:92]: woops, I just copy and pasted from the other file, my bad!

**Pull request merged**